second-hand-b/src/main/java/com/codesquad/secondhand/command/adapter/in/web/chat/ChatCommandController.java

```diff
@@ -1,37 +1,27 @@
 package com.codesquad.secondhand.command.adapter.in.web.chat;
 
 import com.codesquad.secondhand.command.adapter.in.web.chat.request.ChatRoomIdRequest;
-import com.codesquad.secondhand.command.adapter.in.web.chat.response.ChatRoomDetail;
 import com.codesquad.secondhand.command.adapter.in.web.chat.response.ChatRoomId;
-import com.codesquad.secondhand.command.port.in.ChatUseCase;
+import com.codesquad.secondhand.command.port.in.ChatRoomCommandUseCase;
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.ResponseEntity;
 import org.springframework.security.core.annotation.AuthenticationPrincipal;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
 @RequiredArgsConstructor
 @RestController
-public class ChatController {
+public class ChatCommandController {
 
-    private final ChatUseCase chatUseCase;
+    private final ChatRoomCommandUseCase chatRoomCommandUseCase;
 
-    @GetMapping("/api/chats/room-id")
+    @PostMapping("/api/chats/room-id")
     public ResponseEntity<ChatRoomId> getChatRoomId(
             @AuthenticationPrincipal String memberId,
             @RequestBody ChatRoomIdRequest chatRoomIdRequest) {
-        ChatRoomId chatRoomId = chatUseCase.getChatRoomId(chatRoomIdRequest.getProductId(),
+        ChatRoomId chatRoomId = chatRoomCommandUseCase.getOrCreateChatRoomId(chatRoomIdRequest.getProductId(),
                 chatRoomIdRequest.getSellerId(), Long.parseLong(memberId));
```

Comment:
한줄로 쓰는 것이 가독성이 좋을 까요 ?
한줄보다 코드를 다 빼는것이 가독성이 더 좋다고 생각해요. 
```suggestion
        ProductId productId = chatRoomIdRequest.getProductId();
        SellerId sellerId = chatRoomIdRequest.getSellerId();
        long memberId = Long.parseLong(memberId);
        ChatRoomId chatRoomId = chatRoomCommandUseCase.getOrCreateChatRoomId(productId,
                memberId);
```

Reply:
그리고 getOrCreateChatRoomId() 메서드 명이 역할이 두개라 조금 어색하네요.